HomeWorkDZ/HomeWorkDZ/segmentSelectionViewController.swift

```diff
@@ -0,0 +1,108 @@
+
+import UIKit
+
+class segmentSelectionViewController: UIViewController {
+   
+    @IBOutlet weak var segmentControlButten: UISegmentedControl!
+    
+     private lazy var firstTextField: UITextField = {
+        let oneTextField = UITextField()
+         oneTextField.placeholder = "text 1"
+         oneTextField.backgroundColor = .white
+         oneTextField.frame = CGRect(x: 16, y: 100, width: 100, height: 50)
+         return oneTextField
+    }()
+    
+    private lazy var secondTextField: UITextField = {
+       let twoTextField = UITextField()
+        twoTextField.placeholder = "text 2"
+        twoTextField.backgroundColor = .white
+        twoTextField.frame = CGRect(x: 16, y: 200, width: 100, height: 50)
+        return twoTextField
+   }()
+    
+    private lazy var firstButten: UIButton = {
+        let oneButten = UIButton()
```

Comment:
```suggestion
        let oneButton = UIButton()
```
Поправь везде опечатку